File: Python/uri_1134.py
alcool = 0
gasolina = 0
diesel = 0
while True:
    x = int(input())
    if x == 1:
        alcool += 1
    elif x == 2:
        gasolina += 1
    elif x == 3:
        diesel += 1
    elif x == 4:
        break
    else:
        continue
print('MUITO OBRIGADO')
print('Alcool: %d' % alcool)
print('Gasolina: %d' % gasolina)
print('Diesel: %d' % diesel)

File: Go/uri_1134.go
package main

import (
	"fmt"
)

func main() {
	var x, a, g, d = 0, 0, 0, 0
	for {
		fmt.Scanf("%d", &x)
		if x > 0 && x <= 3 {
			if x == 1 {
				a++
			} else if x == 2 {
				g++
			} else {
				d++
			}
		} else if x == 4 {
			break
		} else {
			continue
		}
	}
	fmt.Printf("MUITO OBRIGADO\n")
	fmt.Printf("Alcool: %d\nGasolina: %d\nDiesel: %d\n", a, g, d)

}
